### DISCUSSION TITLE
Integration with CoreDNS

### DISCUSSION THREAD

**Discussion by networkop** [k8snetworkplumbingwg/whereabouts#217] [Ideas]
This is a duplicate of a question I asked in the NPWG slack. I'm wondering if anyone has done (or thought about) CoreDNS integration with whereabouts (through an external plugin). The idea is to provide a stable hostname for each interface assigned by whereabouts and make CoreDNS resolve those hostnames. The main goal is to simplify interaction with multus+whereabouts interfaces from both inside and outside of a k8s clusters.

**Comment by maiqueb** [comment]
That's overall a good idea.

We could register the DNS names, but we wouldn't be able to "promise" connectivity when those names are used. 

Does this first step make sense to you ?

**Comment by networkop** [comment]
yep, makes sense. connectivity would obviously be the responsibility of an administrator. cool, just wanted to make sure no one has done anything like that before to avoid work duplication. thanks!
